terraform/modules/front_door/load_balancer.tf

```diff
@@ -15,28 +14,27 @@ resource "aws_lb" "this" {
 resource "aws_lb_target_group" "this" {
   name        = var.prefix
   port        = var.application_port
-  protocol    = "HTTP"
+  protocol    = "HTTPS"
   vpc_id      = var.vpc_id
   target_type = "ip"

```

Comment:
The target type is "ip" but we're not actually directly specifying any IP addresses. Do we not want a [`aws_lb_target_group_attachment`](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data-infrastructure/pull/28/files) block to associate it with the container? Or is this redundant because we've defined a `load_balancer` configuration block in the ECS service definition?

Reply:
It's not necessary in this setup - we associate the target group with our service in the ECS service definition. 

Also I think the target group attachment resource would only allow associating with a specific instance or container id, which we don't want.